backend/Perflow/Migrations/20210803162535_RunSeeding.cs

```diff
@@ -0,0 +1,268 @@
+﻿using System;
+using Microsoft.EntityFrameworkCore.Migrations;
+
+namespace Perflow.Migrations
+{
+    public partial class RunSeeding : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
```

Comment:
Don't add data modifications in migrations. Migrations should only change data structure. This approach is not safe for production environment